recipes/elapid/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "elapid" %}
+{% set version = "0.3.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a713f02c5fcaec6cfe584c03cc1faed57dcb837b96d9bd8b21c479cd591164dd
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - descartes >=1.1.0
+    - geopandas >=0.7.0
+    - glmnet  # [py<39 and not win]
```

Comment:
This seems to be the only thing here that makes this not a `noarch: python` recipe. Could you explain what this is about exactly? I.e., how come there is a dependency that is only needed when Python is old? (Might also be nice to add a comment to the recipe so contributors to the recipe in the future do not stumble upon this.

Reply:
it's mostly because the `glmnet` package doesn't include a windows build. I was under the impression from [this issue](https://github.com/civisanalytics/python-glmnet/issues/76) that python 3.9 and above was also unavailable. After checking the [conda-forge page](https://anaconda.org/conda-forge/glmnet) that doesn't appear to be the case, so it looks like this could just be `[not win]`.

this is a statistical modeling package that has some nice-to-have features that are unavailable in `sklearn`. `elapid` works around this by using `glmnet` if it's installed, `sklearn` otherwise. I could technically drop the requirement entirely, but imho it's worth having those nice-to-haves.

Reply:
@earth-chris FYI I did follow up with @civisanalytics to be able to maintain `glmnet` project; No one responded tho

Reply:
yea, i saw that, thanks @amirhessam88. seems unlikely that they're gonna resume maintenance on the package, unfortunately, so future updates would probably have to come from someone who forks it and pushes a new package.